docs/serverless/08-02-create-git-function.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Create a Function from Git Repository
+type: Tutorials
+---
+
+This tutorial shows how you can create a simple "Hello World!" Function from Git repository.
+
+## Steps
+
+Follows these steps:
+
+<div tabs name="steps" group="create-function">
+  <details>
+  <summary label="cli">
+  CLI
+  </summary>
+
+1. Export these variables:
+
+    ```bash
+    export NAME={FUNCTION_NAME}
+    export NAMESPACE={FUNCTION_NAMESPACE}
+    ```
+
+2. Create a GitRepository CR that specifies the Git repository metadata:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha1
+    kind: GitRepository
+    metadata:
+      name: $NAME
+    spec:
+      url: "https://github.com/pPrecel/public-gitops"
+    EOF
+    ```
+   
+>**NOTE** Auth possibility TODO
+
+3. Create a Function CR that specifies the Function's logic:
+
+    ```yaml
+    cat <<EOF | kubectl apply -f -
+    apiVersion: serverless.kyma-project.io/v1alpha1
+    kind: Function
+    metadata:
+      name: $NAME
+    spec:
+      type: git
+      runtime: nodejs12
+      source: $NAME
+      reference: master
+      baseDir: js-handler
+    EOF
+    ```
+
+    >**NOTE** To see full spec, go to the xxx page 
```

Comment:
Go to https://github.com/kyma-project/examples/tree/master/orders-service/function